src/java.base/share/classes/java/lang/Class.java

```diff
@@ -4824,4 +4787,12 @@ private int getClassAccessFlagsRaw() {
     }
 
     private native int getClassAccessFlagsRaw0();
+
+    /**
+     * Return most specific method that matches name and parameterTypes.
+     */
+    Method findMethod(boolean publicOnly, String name, Class<?>... parameterTypes) {
+        PublicMethods.MethodList res = getMethodsRecursive(name, parameterTypes, true, publicOnly);
+        return res == null ? null : getReflectionFactory().copyMethod(res.getMostSpecific());
+    }
```

Comment:
Would you mind moving this up to follow getDeclaredPublicMethods, as these are the two are the method finders exposed via JavaLangAccess.